bugbug/bug_snapshot.py

```diff
@@ -869,9 +874,8 @@ def get_inconsistencies(bugs):
     for bug in bugs:
         try:
             rollback(bug, do_assert=True)
-        except Exception as e:
-            print(bug["id"])
-            print(e)
```

Comment:
We still need to log the actual exception, otherwise it will be hard to know exactly what was wrong

Reply:
Ah I didn't notice you are using `logger.exception`, so we should be good here.